datadog_checks_dev/datadog_checks/dev/tooling/commands/test.py

```diff
@@ -135,6 +137,13 @@ def test(
         test_env_vars[E2E_PARENT_PYTHON] = sys.executable
         test_env_vars['TOX_TESTENV_PASSENV'] += f' {E2E_PARENT_PYTHON}'
 
+    if ddtrace:
+        for env in DDTRACE_OPTIONS_LIST:
+            test_env_vars['TOX_TESTENV_PASSENV'] += f' {env}'
+        # Used for CI app product
+        test_env_vars['TOX_TESTENV_PASSENV'] += ' TF_BUILD BUILD* SYSTEM*'
+        test_env_vars['DD_SERVICE'] = os.getenv('DD_SERVICE', 'datadog-integrations-core')
```

Comment:
```suggestion
        test_env_vars['DD_SERVICE'] = os.getenv('DD_SERVICE', 'ddev-integrations')
```

The datadog prefix does not seem required to me. And we can test non core integrations with it as well